packages/generator-emakinacee-react/generators/app/index.js

```diff
@@ -42,9 +42,26 @@ class AppGenerator extends Generator {
             default: this.options.projectName
         });
 
+        prompts.push({
+            type: 'confirm',
```

Comment:
I changed these prompt types `confirm` as it is designed to handle a boolean type input better.
https://github.com/SBoudrias/Inquirer.js#prompt-types

Reply:
but the return value is still a string, right?

Reply:
ok.. works =) cool.. thx!

Reply:
Cool!
And the `confirm` type returns a boolean.